src/endpoints/event.endpoint.js

```diff
@@ -121,7 +121,7 @@ const deleteEvent = (req, res) => {
         for (let i = 0; events.length > i; i++) {
```

Comment:
Could you use a dictionary where the key is the courseId instead? You wouldn't have to loop through the array.